src/rutujaGDec19/Assignment3/Question10.java

```diff
@@ -0,0 +1,21 @@
+package rutujaGDec19.Assignment3;
+/*Sum of elements of two arrays*/
+public class Question10 {
+	static int calSum(int[] arr1,int[] arr2) {
+		int sum = 0;
```

Comment:
instead of doing this you should create one function which do sum of array element and make **sum**  variable as static, for this kind implementation you have call same method two times

Reply:
work on given comment in next PR